oauth2client/_helpers.py

```diff
@@ -199,12 +199,7 @@ def _add_query_parameter(url, name, value):
     """
     if value is None:
```

Comment:
Does

`return url if value is None else update_query_params(url, {name: value})`

fit? If not, retain the `else:` in the `if:`.